app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -170,16 +206,39 @@ def with_disk_attachments_service(vm)
   end
 
   class_methods do
+    def api3_supported_features
+      []
+    end
+
+    def api4_supported_features
+      [:reconfigure_vms, :snapshots]
```

Comment:
To me it looks these two are new to the codebase right?
Do they reflect / correspond to existing methods?
The reason is, because we want only provider agnostic features defined.